wheel/python/clvm_rs/ser.py

```diff
@@ -115,15 +114,15 @@ def sexp_to_bytes(sexp: CLVMStorage) -> bytes:
 
 
 def _op_read_sexp(
-    op_stack: List[OP_STACK_F],
-    val_stack: List[CLVMStorage],
-    f: BinaryIO,
-    new_pair_f: NEW_PAIR_F,
-    new_atom_f: NEW_ATOM_F,
+        op_stack: List[OP_STACK_F],
+        val_stack: List[CLVMStorage],
+        f: BinaryIO,
+        new_pair_f: NEW_PAIR_F,
+        new_atom_f: NEW_ATOM_F,
```

Comment:
Is this a new version of `black`?

Reply:
all diffs dropped in file, i think it was pycharm being goofy